Stack/Opc.Ua.Core/Types/Utils/HiResClock.cs

```diff
@@ -20,47 +20,40 @@ namespace Opc.Ua
     /// <summary>
     /// Produces high resolution timestamps.
     /// </summary>
-    public class HiResClock
+    public static class HiResClock
     {
-        /// <summary>
-        /// Returns the current UTC time (bugs in HALs on some computers can result in time jumping backwards).
-        /// </summary>
-        public static DateTime UtcNow
+        // In Net Core 3.0+ has monotonic timer from system start.
+        // https://docs.microsoft.com/ru-ru/dotnet/api/system.environment.tickcount64?view=netcore-3.1
+#if NETCORE3_0 || NETCORE3_1
```

Comment:
could this also be NETSTANDARD2_1?

Reply:
no, in NETSTANDARD2_1 this property not exist...

Reply:
NETCORE3 never becomes true in a class lib...